parquet-tools/README.md

```diff
@@ -99,9 +99,10 @@ where option is one of:
     -c,--column <arg>  Dump only the given column, can be specified more than
                        once
     -d,--disable-data  Do not dump column data
-       --debug         Disable color output even if supported
+       --debug         Enable debug output
```

Comment:
This text in the README is just the output of running `java -jar parquet-tools.jar -h`, and that help string is generated in `Main.java` (it merges the common option flags like `--debug` into every other command, see https://github.com/apache/parquet-mr/blob/master/parquet-tools/src/main/java/org/apache/parquet/tools/Main.java#L94) - so this help text is auto generated 

I've updated the others items in the README with the correct output.